rplugin/python3/magma/magmabuffer.py

```diff
@@ -78,8 +78,10 @@ def deinit(self):
     def interrupt(self) -> None:
         self.runtime.interrupt()
 
-    def restart(self) -> None:
+    def restart(self, delete_outputs: bool=False) -> None:
         self.runtime.restart()
+        if delete_outputs:
+            self.outputs = {}
```

Comment:
Might need to refresh the interface as well here.

Besides that, I *think* that this is enough to clear the outputs?

Reply:
It did clear on the next tick, I explicitly clear it now, so it clears immediately.